modules/service/src/main/resources/db/migration/V0885__group_system.sql

```diff
@@ -0,0 +1,14 @@
+
+alter table t_group
+add column c_system boolean  not null default false;
+
+-- need to re-create this view to pick up the new column
+drop view v_group;
+create view v_group as
+  select *,
+  case when c_min_interval is not null then c_group_id end as c_min_interval_id,
+  case when c_max_interval is not null then c_group_id end as c_max_interval_id
+  from t_group;
+
+create unique index unique_system_group_name on t_group (c_program_id, c_name) where c_system is true;
```

Comment:
Is the idea to look up system groups by specific names?  Will there be a lot of different kinds -- maybe one per calibration type?  I suppose we could add an enum with the options, along with a value for user-manipulated groups and convert `system` to the enum.  Maybe this was already discussed and decided, which is fine if so.

Reply:
I expect a single calibration group with all of them combined. This `index` is to ensure that the first group we find with a specific name is the one we want.